crates/proof-of-sql/src/sql/proof/query_proof.rs

```diff
@@ -52,6 +52,8 @@ fn get_index_range(
 pub struct QueryProof<CP: CommitmentEvaluationProof> {
     /// Bit distributions
     pub bit_distributions: Vec<BitDistribution>,
+    /// One evaluation lengths
+    pub one_evaluation_lengths: Vec<usize>,
```

Comment:
The proof expressions, at the very least, shouldn't need to produce/consume the length, since they are all determined by the table they are on. So, perhaps the one_evaluation should be passed into verifier_evaluate for the proof expressions, and possibly into the proof plans as well.)

The reason for this is that most of these don't need to be part of the proof since they are determined by the child nodes.

Reply:
@JayWhite2357 I agree. The only place where one_eval probably needs to be a part of the proof is the output one used in `FilterExec`. Even in that case what we really need is records of table lengths.